prow/cmd/deck/main.go

```diff
@@ -408,6 +431,164 @@ func handleBadge(ja *jobs.JobAgent) http.HandlerFunc {
 	}
 }
 
+// handleRequestProwJobViews pre-renders a Spyglass page for a Prowjob source
+// A valid Prow job view url takes the form:
+//
+// /view/prowjob/<jobname>/<buildID>
+//
+// Example:
+// - /view/prowjob/echo-test/1021530234601607168
+func handleRequestProwJobViews(sg *spyglass.Spyglass, ca *config.Agent) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		setHeadersNoCaching(w)
+		start := time.Now()
+		src := strings.TrimPrefix(r.URL.Path, "/view/")
+		fmt.Fprint(w, renderSpyglass(sg, ca, src))
+		elapsed := time.Since(start)
+		logrus.Info("Time taken to return Prowjob view page: ", elapsed)
```

Comment:
Do we really want all this perf data logged at info? Seems like this would be better as debug or else as something exported to velodrome

Reply:
This was something mentioned by steve on another PR, there is a TODO to export these metrics somewhere, I created #8956 for it.